stacks/rio-autoscaler-stack.yaml

```diff
@@ -1,7 +1,7 @@
 configs:
   config-autoscaler:
     enable-scale-to-zero: "true"
-    max-scale-up-rate: "10"
+    max-scale-up-rate: "1000"
```

Comment:
The knative fix looked like a float, any reason we aren't using a float here?

Reply:
Nope, we can. They specifically do float to show that it's a rate, so you can do like `1.1` to barely scale, but they've had it both ways in past.